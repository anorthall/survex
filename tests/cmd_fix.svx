; pos=yes warn=0
*fix A reference ; generates an info diagnostic
*fix B reference 0 0 0
*fix C reference 0 0 0 1.0
*fix D reference 0 0 0 1.0 2.0
*fix E reference 0 0 0 1.0 2.0 3.0
*fix F reference 0 0 0 1.0 2.0 3.0 3.14 2.18 1.44
