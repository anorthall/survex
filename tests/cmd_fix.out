./cmd_fix.svx:2:18: info: FIX command with no coordinates - fixing at (0,0,0)
 *fix A reference ; generates an info diagnostic
                  ^

Removing trailing traverses...

Concatenating traverses...

Simplifying network...

Calculating network...

Calculating traverses...

Calculating trailing traverses...

Calculating statistics...

Survey contains 6 survey stations, joined by 0 legs.
There are 0 loops.
Survey has 6 connected components.
Total length of survey legs =    0.00m (   0.00m adjusted)
Total plan length of survey legs =    0.00m
Total vertical length of survey legs =    0.00m
   6 0-nodes.
